MANUAL_GCP_DEPLOYMENT.md

```diff
@@ -0,0 +1,30 @@
+# Deployment Guide for membrane-backend
+
+This guide outlines the steps to deploy manually membrane-backend to Google Cloud Run (GCP). It uses secrets as volume which allows you to mount your secret keys directly into the service's filesystem, which gives a secure and effective way to manage the application's sensitive information.
+
+This backend application needs to manage keys for communcation with different clients here is a breakdown
+- Server Key Pair:
+
+    * Private Key: Used by the server to decrypt information encrypted with its public key. Must be stored securely and never exposed.
+
+    * Public Key: Shared with clients or other entities that need to encrypt data intended only for the server.
+Client Key Pairs:
+
+- Private Key: Remains with the client. Used to decrypt data or sign transactions.
+- Public Key: Shared with the server and possibly other clients. The server uses this to encrypt data specifically for that client or verify its signature.
+
+## Steps
+ 1. Setup your GCP environment. You can use the instructions [GCP Project Setup Guide](https://github.com/ai-cfia/devops/blob/main/gcp-setup-script/gcp-project-setup-guide.md)
+
+2.  You will need to add the "Secret Manager Secret Accessor" role to your service account
+
+3. Enable ["Secret Manager"](https://cloud.google.com/secret-manager) service on your GCP project.
+
+4. After running the script from the [ReadMe](README.md) file, you should have 4 keys on your project. If you have the public key(s) from your client(s), you can go ahead and delete the one you generated and replace it with the one(s) you have. In the "Secret Manager", create a new secret and for each key, give it the right name and upload it.
+
+5. On Cloud Run, click on the name of the application you want to mount your secrets as volume on to and click on "Edit & Deploy New Revision" on the top of the screen.
+
+<img width="649" alt="Screenshot_58" src="https://github.com/ai-cfia/membrane-backend/assets/9827730/b2b2f4b5-4cb6-4bf5-bc15-519c25d06204">
```

Comment:
I don't think this is a good idea. another absolute URL, this time to an assets branch. The image will break as soon as you rename the repo (for example)

Check in the image and use a relative URL

Reply:
done